languages/java/exercises/concept/case-statement/.docs/instructions.md

```diff
@@ -0,0 +1,29 @@
+You are developing a system to help the staff of a football/soccer club's web site report on matches. Data is received from a variety of sources and piped into a single stream after being cleaned up.
+
+### 1. Output descriptions of the players based on their shirt number
+
+The team only ever plays a 4-3-3 formation and has never agreed with the 1965 change to the rules allowing for substitutions, never mind enlarged squads.
+
+The player descriptions are as follows:
+
+```
+1 -> "goalie"
+2 -> "left back"
+3 & 4 "center back"
+5 -> "right back"
+6, 7, & 8 -> "midfielder"
+9 -> "left wing"
+10 -> "striker"
+11 -> "right wing"
+```
+
+Implement the static `PlayAnalyzer.onField()` method to output a player description based on their shirt number.
```

Comment:
These instructions contain references to three methods:

- `PlayAnalyzer.onField()`
- `PlayAnalyzer.analyzeOnField()`
- `PlayAnalyzer.offField()`

Looking at the source, it looks like there should only be one. Could you choose one and apply that everywhere (this file, example implementation, stub implementation and tests files? My vote goes to the first option BTW: `PlayAnalyzer.onField()`